react-ui/src/components/modifyOrder/ModifyOrderForm.js

```diff
@@ -0,0 +1,195 @@
+import React from 'react'
+import { FormattedMessage, injectIntl } from 'react-intl'
+import '../Style.css'
+import { getFromOpenElisServer } from '../utils/Utils'
+
+import {
+  Heading,
+  Form,
+  FormLabel,
+  TextInput,
+  Button,
+  Grid,
+  Column,
+  DatePicker,
+  DatePickerInput,
+  RadioButton,
+  RadioButtonGroup,
+  Stack,
+  Section,
+
+} from '@carbon/react'
+
+import { patientSearchHeaderData } from '../data/PatientResultsTableHeaders'
+import { Formik, Field } from 'formik'
+import SearchPatientFormValues from '../formModel/innitialValues/SearchPatientFormValues'
+
+class ModifyOrderForm extends React.Component {
+  constructor (props) {
+    super(props)
+    this.state = {
+      dob: '',
+      patientSearchResults: [],
+      page: 1,
+      pageSize: 5
+    }
+  }
+
+  handleSubmit = (values) => {
+    values.dateOfBirth = this.state.dob
+    const searchEndPoint = '/rest/patient-search-results?' + 'lastName=' + values.lastName + '&firstName=' + values.firstName + '&STNumber=' + values.patientId + '&subjectNumber=' + values.patientId + '&nationalID=' + values.patientId + '&labNumber=' + values.labNumber + '&dateOfBirth=' + values.dateOfBirth + '&gender=' + values.gender
+    getFromOpenElisServer(searchEndPoint, this.fetchPatientResults)
+  }
+
+  fetchPatientResults = (patientsResults) => {
+    patientsResults.forEach(item => item.id = item.patientID)
+    // console.log(JSON.stringify(patientsResults))
+    this.setState({ patientSearchResults: patientsResults })
+  }
+
+  fetchPatientDetails = (patientDetails) => {
+    console.log(JSON.stringify(patientDetails))
+    this.props.getSelectedPatient(patientDetails)
+  }
+
+  handleDatePickerChange = (...e) => {
+    this.setState({
+      dob: e[1]
+    })
+  }
+
+  patientSelected = (e) => {
+    const patientSelected = this.state.patientSearchResults.find((patient) => {
+      return patient.patientID == e.target.id
+    })
+    const searchEndPoint = '/rest/patient-details?patientID=' + patientSelected.patientID
+    getFromOpenElisServer(searchEndPoint, this.fetchPatientDetails)
+  }
+
+  handlePageChange = (pageInfo) => {
+    if (this.state.page != pageInfo.page) {
+      this.setState({ page: pageInfo.page })
+    }
+
+    if (this.state.pageSize != pageInfo.pageSize) {
+      this.setState({ pageSize: pageInfo.pageSize })
+    }
+  }
+
+  handlePerPageChange = (newPerPage) => {
+    this.setState({ perPage: newPerPage })
+  }
+
+  render () {
+    const { page, pageSize } = this.state
+    return (
+
```

Comment:
@gitcliff , Looks like you're implementing a Patient search component again here.
we have already written most of these common re-usable components [here](https://github.com/I-TECH-UW/OpenELIS-Global-2/tree/2.8/react-ui/src/components/common). 
you'll need to re-use those components for creating patients , creating orders etc